stress-test.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import os, sys
+import uuid
+import subprocess
+import re
+import time
+from cStringIO import StringIO
+
+from codalab.lib.bundle_cli import BundleCLI, Commands
+from codalab.lib.codalab_manager import CodaLabManager
+from codalab.common import UsageError
+
+from multiprocessing.dummy import Pool as ThreadPool
+
+logger = logging.getLogger(__file__)
+
+class Timer(object):
+    def __enter__(self):
+        self.start = time.clock()
+        return self
+
+    def __exit__(self, eType, eValue, eTrace):
+        self.end = time.clock()
+        self.interval = self.end - self.start
+
+        if eType:
+            return False
+
+class CreateWorksheets(object):
+    def __init__(self, cli, n, multithread=True):
+        self.cli = cli
+        self.n = n
+
+    def __enter__(self):
+        def create_worksheet(i):
+            ws = 'gregwasher91-test-worksheet-{}'.format(i)
```

Comment:
Avoid hardcoding...create an uuid when the script is run to use as a prefix to every name that's used.